index.html

```diff
@@ -10,52 +10,55 @@
 	<link href="https://fonts.googleapis.com/css?family=Open+Sans&amp;subset=latin-ext" rel="stylesheet">
 </head>
 <body>
-<header>
-	<div id="logo-container">
-		<img id="logo" src="img/logo.jpg" alt="Academy Logo" />
+<section id="header">
+    <div class="logo-container">
+        <img class="logo" src="img/logo.jpg" alt="Academy Logo" />
 		<h1>IMG Academy</h1>
-	</div>
-	<nav id="navigation">
-		<ul class="menu">
+    </div>
+</section>
+<section id="navigation">
+    <ul class="menu">
 			<li><a href="#">Home</a></li>
 			<li><a href="#">About</a></li>
 			<li><a href="#">Services</a></li>
 			<li><a href="#">Plans</a></li>
 			<li><a href="#">Contact</a></li>
-		</ul>
-	</nav>
-</header>
-<main>
+    </ul>
+</section>
+<section id="content">
 	<article>
-		<section>
-			<div id="banner-container">
-				<img id="banner" src="img/baner.jpg" alt="Banner" />
-			</div>
-			<div id="description">
+        <div class="banner-container">
+            <img class="banner" src="img/baner.jpg" alt="Banner" />
+        </div>
+        <div class="description">
 				Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-			</div>
-		</section>
+        </div>
 		<aside>

```

Comment:
@nattix Tutaj wszystko Ci się pomieszało :) może ja źle wytłumaczyłem. Zobacz tutaj:
https://codepen.io/tarnasm/pen/MqGNza
To jest napisany przeze mnie w miarę idealny formularz. Zerknij na niego dokładnie. A potem CI go dokładnie opisze co i do czego.